src/test/perf_counters.cc

```diff
@@ -68,6 +68,8 @@ TEST(PerfCounters, SimpleTest) {
   std::string message;
   ASSERT_EQ("", client.do_request("{ \"prefix\": \"perf dump\" }", &message));
   ASSERT_EQ("{}\n", message);
+  ASSERT_EQ("", client.do_request(R"({ "prefix": "counter dump", "format": "raw" })", &message));
```

Comment:
Can this change be made so that the commands look the same and one doesn't need to ponder why `"format": "raw"` is there for the latter but not for the former?
```suggestion
  ASSERT_EQ("", client.do_request("{ \"prefix\": \"counter dump\" }", &message));
```